site/app/templates/course/CourseMaterials.twig

```diff
@@ -8,7 +8,7 @@
 
     <div class="action-buttons" style="float: right; margin-bottom: 20px;">
         {# button to open all of the subfolders and cookie it#}            
-        <a onclick='setCookie("foldersOpen",openAllDivForCourseMaterials());' class="btn btn-primary key_to_click" tabindex="0">Open/Close All Folders</a>
+        <a onclick='Cookies.set("foldersOpen",openAllDivForCourseMaterials());' id="cm_toggle_folders_btn" class="btn btn-primary key_to_click" tabindex="0">Open/Close All Folders</a>
```

Comment:
As per our [style guide](https://submitty.org/developer/software_and_system_design/coding_style_guide/css#styling), this ID should have the words separated by dashes instead of underscores.  If it's challenging to convert this to follow the style guide, please feel free to mark this comment as resolved.